### PR TITLE
OVOWEB-32: Add initial web styles

### PR DESCRIPTION
**Pull request by BrandonWilliamsCS**
Future iterations to come.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/krushtech/style-guide/pull/11" target="_blank">ReviewNinja

**Comment by BrandonWilliamsCS** [comment]
I plan on adding ideas from https://github.com/Platypi/style_typescript and https://basarat.gitbooks.io/typescript/content/docs/styleguide/styleguide.html#null-vs-undefined later.

File: web/krush-markup-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+
+    ```jsx
+    // bad
+    <Foo className="stuff"></Foo>
+
+    // good
+    <Foo className="stuff" />
+    ```
+
+  - If your component has multi-line properties, close its tag on a new line. (Disputed)
```
Comment (kprenger) [inline web/krush-markup-style-guide.md]: I'd agree on this being disputed. Personally, I prefer the "bad" syntax. But I'm not a big web dev, so whatever the community agrees is more common.

**Comment by lukemoderwell** [comment]
!star

File: web/krush-markup-style-guide.md (hunk 1 above)
Comment (lukemoderwell) [inline web/krush-markup-style-guide.md]: Maybe we can have a compromise and use the "bad" syntax when the element is not self-closing while using the "good" with self-closing elements.

**Comment by kprenger** [comment]
!star

**Pull request merged**